src/spring-cloud/azext_spring_cloud/buildpacks_binding.py

```diff
@@ -0,0 +1,80 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=wrong-import-order
+import shlex
+from ._enterprise import DEFAULT_BUILD_SERVICE_NAME
+from ._exception_enterprise import BuildpacksBindingOperationError
+from .vendored_sdks.appplatform.v2022_05_01_preview import models
+
+
+def buildpacks_binding_create(cmd, client, resource_group, service,
+                              name, type, properties=None, secrets=None):
+    _assert_binding_not_exists(client, resource_group, service, name)
+    binding_resource = _build_buildpacks_binding_resource(type, properties, secrets)
+    return _create_or_update_buildpacks_binding(client, resource_group, service, name, binding_resource)
+
+
+def buildpacks_binding_set(cmd, client, resource_group, service,
+                           name, type, properties=None, secrets=None):
+    _assert_binding_exists(client, resource_group, service, name)
+    binding_resource = _build_buildpacks_binding_resource(type, properties, secrets)
+    return _create_or_update_buildpacks_binding(client, resource_group, service, name, binding_resource)
+
+
+def buildpacks_binding_show(cmd, client, resource_group, service, name):
+    binding_resource = _get_buildpacks_binding(client, resource_group, service, name)
+    if binding_resource is None:
+        raise BuildpacksBindingOperationError("Not exist")
```

Comment:
raise CLIError. Dont create exception in CLI

Reply:
Sure